src/main/java/edu/byu/cs/controller/SubmissionController.java

```diff
@@ -311,4 +314,37 @@ public static String getRemoteHeadHash(String repoUrl) {
             throw new RuntimeException(e);
         }
     }
+
+    public static Route submissionsReRunPost = (req, res) -> {
+        reRunSubmissionsInQueue();
```

Comment:
@pawlh @DallinFromEarth Are we concerned about validating the authorization of users? Right now, this request will be accepted regardless of whether or not the the user is logged in, or defined as a admin.

Is there already a pretty straight-forward way to confirm that status before continuing with some functionality? I would expect a method like `verifyAdmin()` etc to be easily accessible.

Reply:
This shouldn't actually be an issue, as the endpoint this is hitting is protected on the backend. non-admins trying this will get a 401

Reply:
@pawlh Can you show me where that happens in the code? (I'm still finding my way around the codebase.)

Reply:
Yup, it is happening where the routes are being registered

https://github.com/softwareconstruction240/autograder/blob/3c5d1098d31363ad3357c53f3fcb0c0aca851808/src/main/java/edu/byu/cs/server/Server.java#L46